pyomo/repn/tests/test_standard.py

```diff
@@ -4010,9 +4010,9 @@ def test_linear1(self):
 
         m.v[1].fixed=True
         rep = generate_standard_repn(e, compute_values=True)
-        self.assertEqual(str(rep.to_expression()), "2 + v[3] + 2*v[4] + 3*v[5]")
+        self.assertEqual(str(rep.to_expression()), "-1 + v[3] + 2*v[4] + 3*v[5]")
         rep = generate_standard_repn(e, compute_values=False)
-        self.assertEqual(str(rep.to_expression()), "v[1] + v[1] + v[3] + 2*v[4] + 3*v[5]")
+        self.assertEqual(str(rep.to_expression()), "v[1] - 2*v[1] + v[3] + 2*v[4] + 3*v[5]")
```

Comment:
Unless I'm mistaken, this test was asserting that the behaviour was wrong.

If the expression is `- v[1] + v[3] + 2*v[4] + 3*v[5]` and we fix `v[1]` to `1` then I can't see why the constant term is `2`.

Reply:
@ruaridhw I agree.